nixos/tests/nar-serve.nix

```diff
@@ -41,7 +42,7 @@ import ./make-test-python.nix (
       drvHash = drvName.split("-")[0]
       server.wait_for_unit("nar-serve.service")
       server.succeed(
-          "curl -o hello -f http://localhost:8383/nix/store/{}/bin/hello".format(drvHash)
+          "curl -o hello -f http://localhost:8383/{}/bin/hello".format(drvHash)
```

Comment:
@rizary @kalbasit is this correct?

Reply:
The test is still failing on master. The path
```python
"http://localhost:8383/nix/store/{}/bin/hello".format(drvHash)
```
still doesn't work but
```python
"http://localhost:8383/store/{}/bin/hello".format(drvHash)
```
and
```python
"http://localhost:8383/{}/bin/hello".format(drvHash)
```
do both work. I'm not sure which path is supposed to be the correct one.

cc @Rizary @kalbasit (as you were pinged by @roberth before)

Reply:
I think the 3rd one is correct.

Reply:
You'd want the first one to work though, because with it, things like documentation with absolute path links just work.
In a web context `href="/nix/store/.../index.html"` leads to that url pattern.

I think around the Nix 2.4 release sort of time frame Nix's internal store path representation dropped the storeDir part, so perhaps this regression is a remnant of that change?

Reply:
Isn't that a bug in nar-serve rather than this test?